install/install.go

```diff
@@ -284,6 +285,22 @@ func checkPods(c *k8s.Client, o Options, i bool) {
 			break
 		}
 	}
+	// add annotation for apparmor
+	if !o.AnnotateExisting {
```

Comment:
We should verify this post KubeArmor starts running I guess? This way we are sure that KubeArmor is running as well and only then we show the message?